orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/MessageHandler.kt

```diff
@@ -37,6 +38,11 @@ interface MessageHandler<M : Message> : (Message) -> Unit {
       throw IllegalArgumentException("Unsupported message type ${message.javaClass.simpleName}")
     }
 
+  fun Collection<ExceptionHandler>.shouldRetry(ex: Exception, taskName: String?): ExceptionHandler.Response? {
```

Comment:
I was just looking at this & I don't think `taskName` can ever be null. IntelliJ probably just made it that way when I extracted the method because the parameter is not annotated.